command/config/config.go

```diff
@@ -97,6 +97,7 @@ type (
 		Tag           string `json:"tag,omitempty" yaml:"tag"`
 		AuthToken     string `json:"auth_token,omitempty" yaml:"auth_token"`
 		GroupID       string `json:"group_id,omitempty" yaml:"group_id"`
+		Disk          string `json:"disk,omitempty" yaml:"disk"`
```

Comment:
we will pass it from pool yaml?

Reply:
yes, and if we don't it will go with the default.

Reply:
what is the size that we are planning to put in prod?

Reply:
SInce we just create one vm in a machine, we can use max memory available. also what is the default size right now?

Reply:
default for sonoma is 128 and ventura is 256. I plan to put both with 256.